dev/ci/ci-equations.sh

```diff
@@ -5,4 +5,8 @@ ci_dir="$(dirname "$0")"
 
 git_download equations
 
+export COQEXTRAFLAGS='-native-compiler no'
+# examples/Makefile.local is overriding COQFLAGS (thus ignoring COQEXTRAFLAGS)
+sed -i.bak 's/\(COQFLAGS =.*\)/\1 -native-compiler no/' \
+  ${CI_BUILD_DIR}/equations/examples/Makefile.local
```

Comment:
CC @mattam82

Reply:
I guess the upstream fix is to `COQEXTRAFLAGS+=....` in `Makefile.coq.local`

Reply:
I fixed the Makefile as well to not override COQFLAGS anymore but extend it, that should do it.

Reply:
Unfortunately not, you should extend COQ*EXTRA*FLAGS, if you set something to COQFLAGS, this overrides everything else (c.f. `COQFLAGS?=-q $(OTHERFLAGS) $(COQEXTRAFLAGS)` in coq_makefile generated makefiles).

Reply:
I see, I fixed it properly this time, and checked that COQFLAGS is not overriden anywhere else in equations.

Reply:
Thanks!